utility/RPi/interrupt.h

```diff
@@ -0,0 +1,57 @@
+/*
+Interrupts functions extruded from wiringPi library by Oitzu.
+
+wiringPi Copyright (c) 2012 Gordon Henderson
+https://projects.drogon.net/raspberry-pi/wiringpi
+wiringPi is free software: GNU Lesser General Public License
+see <http://www.gnu.org/licenses/>
+*/
+
+#include "RF24_arch_config.h"
+
+#define INT_EDGE_SETUP          0
+#define INT_EDGE_FALLING        1
+#define INT_EDGE_RISING         2
+#define INT_EDGE_BOTH           3
+
+/*
+ * interruptHandler:
+ *      This is a thread and gets started to wait for the interrupt we're
+ *      hoping to catch. It will call the user-function when the interrupt
+ *      fires.
+ *********************************************************************************
+ */
+static void *interruptHandler (void *arg);
```

Comment:
One thing I noticed was a warning when compiling: ``` warning: ‘void* interruptHandler(void*)’ declared ‘static’ but never defined [-Wunused-function] ``` which I was able to get rid of by changing the function in the interrupt.h and .c files. from ``` static void ``` to just ``` void ``` which still seems to work ok.